.github/workflows/integrate.yaml

```diff
@@ -23,6 +23,7 @@ jobs:
         with:
           credentials: "${{ secrets.CHARMCRAFT_AUTH }}"
           charm-path: test-charm/
+          upload-image: "true"
```

Comment:
Please add a log output equivalent to `echo "::warning::No resources where uploaded as part of this build"` whenever `upload-image` is set to false.

Reply:
Done. Hope it works!

<img width="1527" alt="Screenshot 2022-01-25 at 11 48 24" src="https://user-images.githubusercontent.com/32944505/150971832-7e1bbc99-a8cf-4826-9105-763b3299e5dc.png">